client/internal/models/secrets.go

```diff
@@ -0,0 +1,27 @@
+// Package models includes client models
+package models
+
+import (
+	"time"
+
+	pb "github.com/mkokoulin/secrets-manager.git/client/internal/pb/secrets"
+)
+
+type Secret struct {
+	ID string `json:"id"`
+	CreatedAt time.Time `json:"created_at"`
+	Type string `json:"type"`
+	IsDeleted bool `json:"is_deleted"`
+	Value map[string]string `json:"value"`
+}
+
+func (s *Secret) TransferValueData() []*pb.Data {
+	var result []*pb.Data
```

Comment:
```suggestion
	result := make([]*pb.Data, 0, len(s.Value))
```
если сразу известна длина массива, выделяй под него необходимый capacity, тогда append работает без переаллокаций памяти.